packages/bpk-component-calendar/src/composeCalendar.js

```diff
@@ -43,10 +43,12 @@ const composeCalendar = (Nav, GridHeader, Grid, CalendarDate) => {
       maxDate,
       minDate,
       month,
+      nextMonthLabel,
```

Comment:
For both of these labels, I attempted to follow the same pattern as `changeMonthLabel` the only difference being that they are optional.